src/client/slack_client.ts

```diff
@@ -1,4 +1,4 @@
-import { WebClient, WebAPICallResult, WebAPICallError } from "@slack/client"
+import { WebClient, RTMClient, WebAPICallResult, WebAPICallError } from "@slack/client"
```

Comment:
RTMClient is never used in this class, 
I think we could remove it @gigihiski

Reply:
alright mas @wahyusumartha i think we can remove it